java/src/main/kotlin/io/spine/validation/java/generate/OptionApplicationCode.kt

```diff
@@ -37,7 +37,7 @@ import io.spine.protodata.java.MethodDeclaration
  * @property fields Additional class-level fields required by the validation logic.
  * @property methods Additional class-level methods required by the validation logic.
  */
-internal class FieldOptionCode(
+internal class OptionApplicationCode(
```

Comment:
This name is 1) long 2) a bit confusing in the sense "it is a code of an option which is passed in the command line of the application". And there are other permutations of confusions around this name.

Can you suggest alternatives, please?